libs/ast_generic/AST_generic.ml

```diff
@@ -1402,8 +1402,8 @@ and pattern =
   | PatList of pattern list bracket
   | PatKeyVal of pattern * pattern (* a kind of PatTuple *)
   (* special case of PatId, TODO: name PatAny *)
-  | PatUnderscore of tok
-  (* OCaml and Scala *)
+  | PatWildcard of tok
+  (* e.g. `_` in OCaml and Scala, `...` in C++ *)
```

Comment:
please keep the comment about OCaml and Scala that was about PatDisj,
and move the comment about PatWildcard before.

Reply:
Done.